.github/workflows/build.yml

```diff
@@ -1,45 +1,36 @@
 name: build
-
 on:
-   push:
-      branches: [master]
-   pull_request:
-      branches: [master]
-
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
 env:
-   CARGO_TERM_COLOR: always
-
+  CARGO_TERM_COLOR: always
+  RUSTFLAGS: "-Dwarnings"
```

Comment:
this is needed to fail ci, see [the clippy ci docs](https://doc.rust-lang.org/stable/clippy/continuous_integration/github_actions.html)